content/docs/1.0.x/reference/api/index.md

```diff
@@ -59,7 +62,19 @@ The Keptn API is documented in terms of a [Swagger API documentation](https://sw
         link="./assets/swagger-example.png"
         caption="Keptn Swagger API documentation - Example"
         width="700px">}}
-
+        
+## Uploading an API file to a Keptn project 
+You can upload an API file to a project in any of the following ways :
+* Use the [keptn add-resource](../../cli/commands/keptn_add-resource) command.
+Note that only some APIs are supported by this command.
+* Upload the file directly to your Git repo.
+* Use the `resource-service` API for uploading
+using the Swagger API as shown above.
+This gives you sections for uploading resources
+on the project, stage or service level.
+
+  To mimic what the `keptn add-resource` command does,
+  use the `POST API` call.
 ## Architecture Details of Keptn API
 
 Keptn 0.6.2 introduced an NGINX as new K8s deployment and service called `api-gateway-ngnix` shown in the figure below. This NGINX allows to route the traffic and ensures that all requests are authenticated using the `/auth` endpoint of the `api-service`.
```

Comment:
Keptn 0.6.2 was a **very** long time ago (almost 3 years ago). I don't think this is required information anymore.

Reply:
Okay, @agardnerIT. I will remove that.

Reply:
Not so fast!  The ancient history references aren't needed but what it describes is still relevant!  I'll suggest a rewrite.  And note that this comment is not related to anything @Bhavya wrote -- just a sign that all this doc is overdue for a review ;-)